<launch>

  <!-- roslaunch zed_wrapper zed2.launch -->
  <!-- roslaunch detectron2_ros detectron2_srv.launch -->
  <!-- roslaunch detectron_extract_3d detectron_extract_3d.launch -->

  <!-- Launch the skeleton extractor node -->
  <node name="pcl_serializer" pkg="pcl_serializer" type="pcl_serializer"  output = "screen">
    <!-- <remap from="input_pointcloud" to="/zed2/zed_node/point_cloud/cloud_registered"/> -->
    <remap from="input_depth" to="/zed2/zed_node/depth/depth_registered"/>
    <remap from="input_image" to="/zed2/zed_node/rgb/image_rect_color/compressed"/>
    <!-- <remap from="input_camera_info" to="/zed2/zed_node/depth/camera_info"/> -->
    <!-- <remap from="transfer_topic" to="/transfer_pcl"/> -->
  </node>

  <!-- <node name="pcl_deserializer" pkg="detectron2_pcl_sync" type="pcl_deserializer"  output = "screen">
    <remap from="input_image" to="/result_image"/>
    <remap from="result_image" to="/test_depth"/>
  </node> -->

 
</launch>

